packages/cli/src/commands/site/deploy/index.ts

```diff
@@ -117,7 +117,13 @@ const command: CommandModule<
   },
   handler: async (args) => {
     const command = await import("./siteDeployCommand.mjs");
-    await command.default(args);
+    const selectedVersion = args.version != null
+      ? args.version
+      : {
+        type: args.autoVersion as "git-describe",
+        tagPrefix: args.gitTagPrefix,
+      };
+    await command.default({ ...args, selectedVersion });
```

Comment:
May make debugging slightly easier down the line if no extra information is passed in:

```suggestion
    const { version, autoVersion, gitTagPrefix, ...restArgs } = args;
    await command.default({ ...restArgs, selectedVersion });
```